datasets/lener_br/lener_br.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""LeNER-Br dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+
+import datasets
+
+
+_CITATION = """
+@inproceedings{luz_etal_propor2018,
+    author = {Pedro H. {Luz de Araujo} and Te\'{o}filo E. {de Campos} and
+    Renato R. R. {de Oliveira} and Matheus Stauffer and
+    Samuel Couto and Paulo Bermejo},
+    title = {{LeNER-Br}: a Dataset for Named Entity Recognition in {Brazilian} Legal Text},
+    booktitle = {International Conference on the Computational Processing of Portuguese ({PROPOR})},
+    publisher = {Springer},
+    series = {Lecture Notes on Computer Science ({LNCS})},
+    pages = {313--323},
+    year = {2018},
+    month = {September 24-26},
+    address = {Canela, RS, Brazil},
+    doi = {10.1007/978-3-319-99722-3_32},
+    url = {https://cic.unb.br/~teodecampos/LeNER-Br/},
+}
+"""
+
+_DESCRIPTION = """
+LeNER-Br is a Portuguese language dataset for named entity recognition
+applied to legal documents. LeNER-Br consists entirely of manually annotated
+legislation and legal cases texts and contains tags for persons, locations,
+time entities, organizations, legislation and legal cases.
+To compose the dataset, 66 legal documents from several Brazilian Courts were
+collected. Courts of superior and state levels were considered, such as Supremo
+Tribunal Federal, Superior Tribunal de Justiça, Tribunal de Justiça de Minas
+Gerais and Tribunal de Contas da União. In addition, four legislation documents
+were collected, such as "Lei Maria da Penha", giving a total of 70 documents
+"""
+
+_HOMEPAGE = "https://cic.unb.br/~teodecampos/LeNER-Br/"
+
+_URL = "https://github.com/peluz/lener-br/raw/master/leNER-Br/"
+_TRAINING_FILE = "train/train.conll"
+_DEV_FILE = "dev/dev.conll"
+_TEST_FILE = "test/test.conll"
+
+
+class LeNERBr(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class LenerBr(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `lener_br`